### ISSUE TITLE
No Html-file generated

### ISSUE THREAD

**Issue by jerepaajanen** [rejas/gulp-favicons#2]
Since 2.2.7 to 2.3.0 update, html generation has stopped working.

**Comment by rejas** [comment]
Can NOT confirm this, still works on my project. Can you post a little more info on the config/environement you are using? Or if there are errors/warnings etc?

**Comment by jerepaajanen** [comment]
Sure, this is the gulp task i use:

gulp.task('images:favicons-generate', function () {

    if (isProduction) {
        fs.writeFileSync(paths.src + 'partials/favicons.html', '');

        return gulp.src(paths.images.src + '/favicon.png')
            .pipe($.favicons({
                appName: config.siteTitle,
                appDescription: config.siteDescription,
                url: config.siteURL,
                developerName: pkg.author,
                developerURL: pkg.homepage,
                background: config.faviconsBackground,
                path: config.faviconsPath,
                display: 'standalone',
                orientation: 'portrait',
                version: 1.0,
                logging: false,
                online: false,
                html: paths.src + 'partials/favicons.html'
            }))
            .pipe($.imagemin())
            .pipe(gulp.dest(paths.images.dest + 'favicons'));

    }

});

I tried with logging true, it says that html file edited but there's no error and favicons.html is zero bytes after task has run.

**Comment by rejas** [comment]
Well, I still cant get it reproduced, tried to adapt your config into my build pipeline like this:

```
gulp.task('prepare:favicons', () => {
    return gulp.src('org/favicon.png')
        .pipe(plugins.favicons({
            appName: "config.siteTitle",
            appDescription: "config.siteDescription",
            url: "config.siteURL",
            developerName: "pkg.author",
            developerURL: "pkg.homepage",
            background: '#FFFFFF',
            path: '/favicons',
            display: 'standalone',
            orientation: 'portrait',
            version: 1.0,
            logging: false,
            online: false,
            html: './partials/favicons.html'
        }))
        .pipe(gulp.dest(`${dirs.src}/page/favicons`));
});
```

To me it looks like one of your other steps might be the culprit. Have you tried removing the line with imagemin or the one with writeFileSync?

**Comment by jerepaajanen** [comment]
Tried that too. Set logging to true. I get this message in the end: "[Favicons] Files:create: Creating file: .src/partials/favicons.html..." but no file shows up in that folder.

**Comment by rejas** [comment]
Sorry, cant help you I think. Basically the gulp plugin here is not much more than a thin wrapper around favicon itself. 
Have you tried using the example code from the favicon readme that shows how to call favion from a gulp file without this plugin?

**Comment by jerepaajanen** [comment]
Tried without gulp wrapper. Same problem. Tweaked the "html: " path. Seems like it it tries to output html file only to where I have set gulp.dest, in this case to dist/images/favicons. Something has changed in creating html file, I cannot create it in src/partials/ folder anymone.

html: './src/partials/favicons.html' --> goes inside dist/images/favicons/src/partials/
html: 'favicons.html' --> goes inside dist/images/favicons/

How can I just rewrite favicons.html file in src folder?

What does these do:
pipeHTML: true,
replace: true
